kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProduceAction.java

```diff
@@ -118,6 +120,10 @@ public void produce(
       MappingIterator<ProduceRequest> requests)
       throws Exception {
 
+    if (requests == null) {
+      throw Errors.invalidPayloadException("Null input provided. Data is required.");
+    }
```

Comment:
I'm guessing `null` means a completely empty stream. Would the most consistent way to handle this be just to return empty instead?